src/libraries/System.IO.Pipes/ref/System.IO.Pipes.cs

```diff
@@ -111,16 +111,17 @@ public abstract partial class PipeStream : System.IO.Stream
         public virtual System.IO.Pipes.PipeTransmissionMode ReadMode { get { throw null; } set { } }
         public Microsoft.Win32.SafeHandles.SafePipeHandle SafePipeHandle { get { throw null; } }
         public virtual System.IO.Pipes.PipeTransmissionMode TransmissionMode { get { throw null; } }
-        public override System.IAsyncResult BeginRead(byte[] buffer, int offset, int count, System.AsyncCallback callback, object state) { throw null; }
-        public override System.IAsyncResult BeginWrite(byte[] buffer, int offset, int count, System.AsyncCallback callback, object state) { throw null; }
+        public override System.IAsyncResult BeginRead(byte[] buffer, int offset, int count, System.AsyncCallback callback, object? state) { throw null; }
+        public override System.IAsyncResult BeginWrite(byte[] buffer, int offset, int count, System.AsyncCallback callback, object? state) { throw null; }
```

Comment:
The `callback` should be nullable for both BeginXx methods, too.

Reply:
The System.IO.Stream annotation doesn't admit nullable callbacks:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/IO/Stream.cs#L343
Should we revisit?

Reply:
> Should we revisit?

Yes, that's a bug.

Reply:
(I'll fix them.)